showQuestions.js

```diff
@@ -19,22 +19,27 @@ var converter = new showdown.Converter();
 async function showInstructions() {
   var elements = document.getElementsByTagName("div");
   var noteText = "Upload the questions before you hit start";
-  chrome.storage.local.get("questions", ({questions}) => {
-    if (questions && questions.length>2) {
-      noteText = "There is already a loaded set of questions in the browser cache. Re-upload if you want to be sure, and click start."
-    } 
-    for (item of elements) {
-      try {
-        if (item.className === "cell w99 h10") {
-          item.id = "42";
-          const instructionsElement = getInstructionsElement(noteText);
-          item.append(instructionsElement);
-        }
-      } catch (e) {}
+  const {questions} = await chrome.storage.local.get("questions");
```

Comment:
This "chrome" var:
1. Is it a global provided solely by Chrome browser?
2. Is there a standards compliant alternative? Or at least, webkit equivalent?  

Specially asked about webkit because I think Safari on Mac, edge on windows and chrome are all webkit based.  We could be doing all the "default browser users" a favor by not forcing them to use chrome. 

Don't get me wrong: chrome is great! But it has atrociously poor battery life on laptops, esp crisply MacBooks.  A story I heard from a long time ago: chrome & google's philosophy in general is to offload super heavy computational tasks to the client. In order to juice every bit of perf, they manually set the processor clock to some super small value like 1ns; whereas most browser like IE or Safari set it some 15ns. 

Take the numbers and unit with a grain of salt. 

Anyways, the point is apparently the super fluid, fast experience chrome comes from them taking advantage of sacrificing power/efficiency over speed. Great on desktop! Not so much on laptops. 

I'd bet many laptop users prefer not using chrome for this reason 

Thanks for reading my spiel 🙏🏾🤩

Reply:
I will let the author speak more on this, but my two cents:

 I am guessing since it is a Chrome extension, he used `chrome.local`. It is fully supported by Firefox and Edge. The other `localStorage` is `window.localStorage`. More [here](https://stackoverflow.com/questions/24279495/window-localstorage-vs-chrome-storage-local). It is all part of Web APIs, and if you are learning JS and the async nature of it, is a delight to learn about!

Reply:
ahh I see.  That link is super informative. Delight indeed! Thanks for sharing!